lib/routes/charge/items/create_item.dart

```diff
@@ -160,7 +160,7 @@ class CreateItemPageState extends State<CreateItemPage> {
                                                 return DropdownMenuItem<String>(
                                                   value: value.symbol,
                                                   child: Text(
-                                                    value.displayName,
+                                                    value.symbol.toUpperCase(),
```

Comment:
Maybe instead of `toUpperCase()` we should define the symbol as SAT ?

Reply:
This was causibg issues recognizing older data created with "Sat".
We can do as you say, if we are sure no-one has used this feature yet or we can add code to handle that older data set.

Reply:
This was causing issues recognizing older data created with "Sat".
We can do as you say, if we are sure no-one has used this feature yet or we can add code to handle that older data set.

Reply:
Suggestion:
![image](https://user-images.githubusercontent.com/4012752/76228597-fdd90800-6231-11ea-8ef4-2c8797b47d56.png)

Reply:
I think the suggestion is reasonable. 
Maybe even consider:
`c.symbol.topUpperCase() == symbol.toUpperCase()`
To be on the safe side.

Reply:
[Done](https://github.com/breez/breezmobile/pull/290/commits/7d1fed83dfdf843dd890f7aac4483f8601bf5cf5)